spec/controllers/concerns/administrate/punditize_spec.rb

```diff
@@ -8,9 +8,26 @@ module Administrate
       end
     end
 
+    unless Pundit.const_defined?(:Authorization)
+      # Spec for pundit < 2.2
+      it "exists when Pundit::Authorization is not defined" do
+        reload_punditize
+        expect(Pundit.const_defined?(:Authorization)).to be_falsey
```

Comment:
This assertion is somewhat redundant. The idea is to make it clear that Pundit::Authorization is not defined for punit < 2.2.